package/yast2-core.spec

```diff
@@ -44,6 +44,7 @@ BuildRequires:  libtool
 # we have a parser
 BuildRequires:  bison
 BuildRequires:  flex
+BuildRequires:  jemalloc-devel
```

Comment:
I'd just a comment here that it's a workaround (add the bnc number) so we could possibly remove it later when not needed anymore.